replaying-share/src/main/java/com/jakewharton/rx/ReplayingShare.java

```diff
@@ -35,6 +36,9 @@
  */
 public final class ReplayingShare<T>
     implements ObservableTransformer<T, T>, FlowableTransformer<T, T> {
+
+  private final AtomicReference<T> value;
```

Comment:
This never changes its value so it doesn't need to be atomic. Also please move down below static methods.